pkg/cc/config/config.go

```diff
@@ -12,28 +14,37 @@ import (
 	"github.com/spf13/viper"
 )
 
-var (
-	EnvironmentErr = errors.New("unknown environment")
-)
+const ConfigPath = "config"
 
 // Overrider is a func that mutates configuration.
 type Overrider func(*Config)
 
 type Auth struct {
-	Issuer   string `json:"issuer"`
-	ClientID string `json:"client_id"`
-	Audience string `json:"audience"`
+	Issuer   string `json:"issuer" yaml:"issuer"`
```

Comment:
I don't think you need to add all those `yaml` tags. We don't do it in any of our services.
We just set the [config type to "yaml"](https://github.com/aserto-dev/directory/blob/main/pkg/cc/config/config.go#L188) and [override the tag name to "json"](https://github.com/aserto-dev/directory/blob/main/pkg/cc/config/config.go#L218).

Reply:
I need them when I marshal the config in order to write in on disk https://github.com/aserto-dev/aserto/blob/a2d12a3338d5b2fdc5d62b454bfe8b654c0ab309/pkg/handlers/user/tenant.go#L78

Reply:
I think we can use `viper.WriteConfigAs(path)`: https://github.com/spf13/viper#writing-config-files